build/MSBuildExtensions.targets

```diff
@@ -63,6 +63,8 @@ Copyright (c) .NET Foundation. All rights reserved.
     <WriteLinesToFile File="$(GeneratedMSBuildExtensionsDirectory)/$(BundledVersionsPropsFolder)/$(BundledVersionsPropsFileName)"
                       Lines="$(BundledVersionsPropsContent)"
                       Overwrite="true" />
-
+    <Copy
+      DestinationFiles="$(SdkOutputDirectory)/$(BundledVersionsPropsFileName)"
```

Comment:
(nit) usually it goes copy `source` `destination`.  This tripped me up to put the destination first.